projects/aca-content/viewer/src/lib/components/viewer/viewer.component.html

```diff
@@ -7,12 +7,13 @@
     [nodeId]="nodeId"
     [versionId]="versionId"
     [allowNavigate]="navigateMultiple"
-    [allowRightSidebar]="true"
+    [allowRightSidebar]="false"
```

Comment:
Why 'false'? 
I think this will break 'View Details' button in toolbar because after that change it is still visible but not clickable. 
See videos: 
1. Working button - in remote environment: 

https://github.com/Alfresco/alfresco-content-app/assets/115619721/4c7d8f60-e7e6-46b5-b881-0f8c478f1323

2. Not working button on your branch (right panel is not visible after clicking on button): 

https://github.com/Alfresco/alfresco-content-app/assets/115619721/885a3217-6e9f-4be6-90fa-2f227aa64e16

Reply:
Done

Reply:
Hmm do I understand correctly that you hide this button now by that change?
<img width="852" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/dee91de3-dc7f-43d4-95ee-8cde37d4a46e">


Is not that breaking change? Don't we want to show info panel (properties panel) anymore in file's viewer @AnukritiGL?

cc: @MichalKinas

Reply:
@AleksanderSklorz previously we were hiding the Info button by CSS, but @swapnil-verma-gl suggested to do it by using the input property thats why I implemented it in this way.
Please check the video we can see the property panel we just hiding the button using this check.

[screen-capture (1).webm](https://github.com/Alfresco/alfresco-content-app/assets/106949107/c6c5d5f1-0a57-41fb-b93c-b06dade7dfe1)

Reply:
ok, got it, thanks